css/home-page.css

```diff
@@ -0,0 +1,171 @@
+.home-page{
+    background-color: lightgoldenrodyellow;
+}
+
+.home-header{
+    align-items: center;
+    background-color: gray;
+    background-image: url(background-soft2.jpeg);
+    background-repeat: no-repeat;
+    border: 2px solid black;
+    border-radius: 5px;
+    border-radius: 5px;
+    display: flex;
+    flex-direction: row;
+    flex-wrap: wrap; 
+    gap: 25%;
+    min-height: 160px;
+    justify-content: end;
+}
+
+.header-content{
+    display: flex;
+    flex-direction: row;
+    gap: 50px;
+    padding: 50px 20;
+    justify-content: flex-start;
+
+}
+
+.profile-img{
+    margin-left: 150px;
+}
+
+.navbar-container{
+    background-color: gray;
+    border: 2px solid black;
+    border-radius: 5px;
+    float: left;
+    margin-left: 40px;
+    margin-top: 20px;   
+    width: 200px;
+
+    
+}
+
+.home-navbar {
+    list-style-type: none;
+    padding: 0;
+}
+
+.nav-branch {
+    position: relative;
+}
+
+.nav-links {
+    color: #000;
+    display: block;
+    padding: 10px 15px;
+    text-decoration: none;
+}
+
+.items-text {
```

Comment:
This class isn't used. Remove it.
In general, remove all unused/dead code (classes, HTML, functions ecc...)

Reply:
fixed

Reply:
i've already removed .items-text from my code